newrelic/core/stats_engine.py

```diff
@@ -882,9 +882,15 @@ def notice_error(self, error=None, attributes=None, expected=None, ignore=None,
             if attr.destinations & DST_ERROR_COLLECTOR:
                 attributes["userAttributes"][attr.name] = attr.value
 
+        # Instead of creating a brand new transaction, we
+        # simply attach the entity_guid to the intrinsics
+        # as well as the TracedError
+        no_transaction_guid = getattr(settings, "entity_guid", None)
+
         # pass expected attribute in to ensure we capture overrides
         attributes["intrinsics"] = {
             "error.expected": is_expected,
+            "guid": no_transaction_guid,
```

Comment:
I think we should separate this part of the change until we can confirm 100% what the behavior will be like, taking this to the spec reviewers meeting.